desktop/features.php

```diff
@@ -344,9 +344,9 @@
         <img src="<?php echo cdn("img/developer9.png"); ?>"/>
     </div>
     <div class="section" id="unicode">
-        <h2 class="red underline">Unicode 11.0 Compliant</h2>
+        <h2 class="red underline">Unicode 12.0 Compliant</h2>
```

Comment:
Can I squirm here? Not sure we have updated the unicode character database to 12.0...

Reply:
PR keymanapp/keyman#1834 added the Unicode 12.0 characters.

Reply:
Well there ya go. I approved that PR and all. 😁